js/popup.js

```diff
@@ -0,0 +1,70 @@
+import {createSimilarAds} from './data.js';
+
+const TYPES_RUS = {
+  palace: 'Дворец',
+  flat: 'Квартира',
+  house: 'Дом',
+  bungalow: 'Бунгало',
+  hotel: 'Отель',
+};
+
+const similarAdsTemplate = document.querySelector('#card').content.querySelector('.popup');
+const similarAdsList = document.querySelector('#map-canvas');
+
+const similarAds = createSimilarAds();
+
+const getPopup = (data) => {
+  const adsElement = similarAdsTemplate.cloneNode(true);
+  adsElement.querySelector('.popup__title').textContent = data.offer.title;
+  adsElement.querySelector('.popup__text--address').textContent = data.offer.address;
+  adsElement.querySelector('.popup__text--price').textContent = `${data.offer.price} ₽/ночь`;
+  adsElement.querySelector('.popup__type').textContent = TYPES_RUS[data.offer.type];
+  adsElement.querySelector('.popup__text--capacity').textContent = `${data.offer.rooms} комнаты для ${data.offer.guests} гостей`;
+  adsElement.querySelector('.popup__text--time').textContent = `Заезд после ${data.offer.checkin}, выезд до ${data.offer.checkout}`;
+  const descriptionContainer = adsElement.querySelector('.popup__description');
+  descriptionContainer.textContent = data.offer.description;
+
+  if (!data.offer.description) {
+    descriptionContainer.remove();
+  }
+
+  const featuresContainer = similarAdsTemplate.querySelector('.popup__features');
```

Comment:
Сейчас все работает, только здесь ошибка, должно быть так
```suggestion
  const featuresContainer = adsElement.querySelector('.popup__features');
```